docs/programming/MPIBestpractices.md

```diff
@@ -1,13 +1,13 @@
 # MPI Best practices
-MPI or Message Passing Interface is a powerful library standard that allows for the parallel execution of applications across multiple processors on a system. It differs from other parallel execution libraries like OpenMP by also allowing a user to run their applications across multiple nodes. Unfortunately it can sometimes be a bit tricky to run a compiled MPI application within an HPC resource. The following page outlines best practices in running your MPI applications across CURC resources.  
+MPI, or Message Passing Interface, is a powerful library standard that allows for the parallel execution of applications across multiple processors on a system. It differs from other parallel execution libraries like OpenMP by also allowing a user to run their applications across multiple nodes. Unfortunately it can sometimes be a bit tricky to run a compiled MPI application within an HPC resource. The following page outlines best practices in running your MPI applications across CURC resources.  

```

Comment:
On this page can you put the multiple options or examples in tabs? For example, this part: 
![Screenshot 2024-09-18 at 4 29 47 PM](https://github.com/user-attachments/assets/907291ff-47de-4bdb-84e5-0da4c95557ed)